app/controllers/admin/api/access_tokens_controller.rb

```diff
@@ -12,16 +12,33 @@ class Admin::Api::AccessTokensController < Admin::Api::BaseController
 
   before_action :authorize_access_tokens
 
+  ##~ sapi = source2swagger.namespace("Account Management API")
+  ##~ e = sapi.apis.add
+  ##~ e.path = "/admin/api/users/{user_id}/access_tokens.json"
+  ##~ e.responseClass = "List[access_token]"
+  #
+  ##~ op = e.operations.add
+  ##~ op.httpMethod = "POST"
+  ##~ op.summary   = "Access Token Create"
+  ##~ op.description = "Creates an access token."
+  ##~ op.group = "access_token"
+  #
+  ##~ op.parameters.add :name => "user_id", :description => "ID of the user.", :dataType => "integer", :paramType => "path", :required => true
+  ##~ op.parameters.add :name => "name", :description => "Name of the access token.", :dataType => "string", :required => true, :paramType => "query"
+  ##~ op.parameters.add :name => "permission", :description => "Permission of the access token. It must be either 'ro' (read only) or 'rw' (read and write).", :dataType => "string", :required => true, :paramType => "query"
+  ##~ op.parameters.add :name => "scopes", :description => "Scope of the access token. It is an Array containing one or more of these values: [\"cms\", \"finance\", \"account_management\", \"stats\"]", :dataType => "string", :required => true, :paramType => "query"
```

Comment:
cms is only available to master account on saas, no?

Reply:
Right, this is what you are talking about:
https://github.com/3scale/porta/blob/6d7ee3c6ce333a197b41ecec89af64c2da2867a2/app/models/access_token.rb#L93-L100
The problem is this documentation needs to be pure text, not code. At least as far as I know.
So right now I have them all in the documentation and I allow to set them all, and later the authentication itself takes care of this:
https://github.com/3scale/porta/blob/6d7ee3c6ce333a197b41ecec89af64c2da2867a2/lib/api_authentication/by_access_token.rb#L217-L219
What I don't know is if I should allow to set only the `allowed_ones` and explain in the documentation which ones are allowed depending on the case it is.
What do you think? @thomasmaas

Reply:
I think we should only explain the allowed ones in documentation and if possible add a comment that refers to the cms scope which is not visible in the generated docs

Reply:
![image](https://user-images.githubusercontent.com/11318903/50210493-96a60e00-0376-11e9-98fa-8e86fe802b1e.png)